src/pages/edge/adobe-journey-optimizer-decisioning/api-reference.md

```diff
@@ -116,11 +132,31 @@ Completion callback passed to `updatePropositions` supports network timeout and
 
 Android
 
-<Tabs query="platform=android&api=update-propositions-withError"/>
+<Tabs query="platform=android&api=update-propositions-withCallback"/>
+
+iOS
+
+<Tabs query="platform=ios&api=update-propositions-withCallback"/>
+
+## updatePropositionsWithCompletionHandlerWithCompletionTimeout
```

Comment:
nit: Can we call this `updatePropositionsWithCompletionHandlerAndTimeout to make it a bit succinct

Reply:
done @spoorthipujariadobe